docker/ubuntu_20.04-dev.dockerfile

```diff
@@ -22,8 +22,6 @@ RUN if [ "${py_version%.?}" -eq 3 ] ; \
             libblas-dev \
             liblapack-dev \
             libeigen3-dev \
-            libhdf5-dev \
-            hdf5-tools \
```

Comment:
Keep these here, it should work if they are here and the other problems are fixed.